docs/modules/architecture/pages/vector-search-overview.adoc

```diff
@@ -0,0 +1,52 @@
+= Vector Database Engine
```

Comment:
add this page to navigation

Reply:
https://github.com/hazelcast/hz-docs/pull/1134/commits/1ee1b9a5612fed748bcd23d4787699d4d64ba8c4